docs/user-guide/repository-management.rst

```diff
@@ -0,0 +1,144 @@
+is configured to mirror certain repositories, this feature can be used to associate profiles with those repositories.
+Systems installed under those profiles will be autoconfigured to use these repository mirrors in ``/etc/yum.repos.d``,
+and if supported (Fedora Core 6 and later), these repositories can be leveraged within Anaconda.
+
+This can be useful if
+
+#. you have a large install base, or
+#. you want fast installation and upgrades for your systems, or
+#. have some extra software not in a standard repository but want provisioned systems to know about that repository.
+
+Make sure there is plenty of space in Cobbler's webdir, which defaults to ``/var/www/cobbler``.
+
+.. code-block:: shell
+
+    cobbler reposync [--only=ONLY] [--tries=N] [--no-fail]
+
+``cobbler reposync`` is used to update repos known to Cobbler. The command is required to be executed prior to the first
+provisioning of a system if Cobbler is configured as a mirror. If you just add repos and never run ``cobbler reposync``,
+the content of the repos will be missing. This is probably a command you should include in a crontab. The configuration
+is left up to the systems administrator.
+
+.. note:: Mirroring can take a long time because of the amount of data being downloaded.
+
+For those familiar with dnf's reposync, Cobbler's reposync is mostly a wrapper around the ``dnf reposync`` command.
+use "cobbler reposync" to update Cobbler mirrors, as dnf's reposync does not perform all required steps. Also Cobbler
+adds support for rsync and SSH locations, where as dnf's reposync only supports what yum supports (http/ftp).
+
+If you want to update a certain repository, run:
+
+.. code-block:: shell
+
+    cobbler reposync --only="reponame1" ...
+
+When updating repos by name, a repo will be updated even if it is set to be not updated during a regular reposync
+operation (ex: ``cobbler repo edit --name=reponame1 --keep-updated=False``).
+
+For distributions using dnf/yum Cobbler can act as a mirror and generate the ``.repo`` files for the core system
+packages. This is only possible if the ``cobbler import`` command provided enough information. If this feature is
+desirable, it can be turned on by setting ``yum_post_install_mirror`` to ``True`` in ``/etc/cobbler/settings.yaml`` (and
+running ``cobbler sync``). You should not use this feature if machines are provisioned on a different VLAN/network than
+production, or if you are provisioning laptops that will want to acquire updates on multiple networks.
+
+The flags ``--tries=N`` (for example, ``--tries=3``) and ``--no-fail`` should likely be used when putting reposync on a
+crontab. They ensure network glitches in one repo can be retried and also that a failure to synchronize one repo does
+not stop other repositories from being synchronized.
+
+Importing trees workflow
+########################
+
+Cobbler can auto-add distributions and profiles from remote sources, whether this is a filesystem path or an rsync
```

Comment:
Small typo:
```suggestion
Cobbler can auto-add distributions and profiles from remote sources, whether this is a filesystem path or a rsync
```

Reply:
As far as I understand the [grammar rules](https://www.merriam-webster.com/words-at-play/is-it-a-or-an), "an" is correct here. "A rsync" is hard to pronounce, at least for me.

Reply:
Ahahaha okay I will revert it.